README.md

```diff
@@ -1,28 +1,65 @@
 # OmbuLabs::Auth
-Short description and motivation.
+> _Refactor a rack middleware (probably a ruby gem) out of https://points.ombulabs.com/ so that we can easily add GitHub-based authentication to other Ruby/Rails internal applications._
+>
+> _It would be useful if we could add it to http://skunk.fastruby.io (backend for skunk) so that anybody who is part of our organization could sign in and review details in the admin panel._
+>
+> _Said gem should be easily configurable (only with environment variables) and should make it very simple to add "OmbuLabs-authentication" to our internal Ruby apps._
 
-## Usage
-How to use my plugin.
+## Configs
+Make sure you configure your ENV variables to use Github authentication.
 
+```
+ENV["GITHUB_APP_ID"]
+ENV["GITHUB_APP_SECRET"]
+ENV["ORGANIZATION_LOGIN"]
+```
 ## Installation
-Add this line to your application's Gemfile:
+
+- Clone this gem inside the `lib` folder of your project
+
+
+- Add these lines to your application's Gemfile:
 
 ```ruby
-gem "ombu_labs-auth"
+gem 'ombu_labs-auth', path: 'lib/ombu_labs-auth'
+gem 'omniauth-github', '~> 2.0.0'
```

Comment:
[Because of this](https://github.com/fastruby/ombu_labs-auth/blob/main/ombu_labs-auth.gemspec#L28), I don't think this line is needed ☝️

Reply:
I currently have no idea why yet, but it is necessary. 

```
/Users/casanova/.rbenv/versions/3.1.2/lib/ruby/gems/3.1.0/gems/devise-4.8.1/lib/devise/omniauth/config.rb:42:in `autoload_strategy': Could not find a strategy with name `Github'. Please ensure it is required or explicitly set it using the :strategy_class option. (Devise::OmniAuth::StrategyNotFound)

          raise StrategyNotFound, name
          ^^^^^
```

I added an issue for this - https://github.com/fastruby/ombu_labs-auth/issues/14